### PR TITLE
re-enable fidget

### PR DESCRIPTION
**Pull request by ringabout**

### CODE REVIEW

File: testament/important_packages.nim [hunk 1]
```diff
@@ -54,7 +54,7 @@ pkg1 "delaunay"
 pkg1 "docopt"
 pkg1 "easygl", "nim c -o:egl -r src/easygl.nim", "https://github.com/jackmott/easygl"
 pkg1 "elvis"
-# pkg1 "fidget" # pending https://github.com/treeform/fidget/issues/133
+pkg1 "fidget"
```
Comment (timotheecour) [inline testament/important_packages.nim]:
```suggestion
pkg1 "fidget", useHead = false
```
refs https://github.com/treeform/fidget/issues/133#issuecomment-787532607 ?

**Comment by ringabout** [comment]
Ci failed.

File: testament/important_packages.nim [hunk 2]
```diff
@@ -54,7 +54,7 @@ pkg1 "delaunay"
 pkg1 "docopt"
 pkg1 "easygl", "nim c -o:egl -r src/easygl.nim", "https://github.com/jackmott/easygl"
 pkg1 "elvis"
-# pkg1 "fidget" # pending https://github.com/treeform/fidget/issues/133
+pkg1 "fidget", useHead = false
```
Comment (timotheecour) [inline testament/important_packages.nim]:
```suggestion
pkg1 "fidget", useHead = false # refs https://github.com/treeform/fidget/issues/133
```

to document why we needed `useHead = false`

**Review by timotheecour — APPROVED** [review]

File: testament/important_packages.nim [hunk 3]
```diff
@@ -59,7 +60,7 @@ pkg "delaunay"
 pkg "docopt"
 pkg "easygl", "nim c -o:egl -r src/easygl.nim", "https://github.com/jackmott/easygl"
 pkg "elvis"
-# pkg "fidget" # pending https://github.com/treeform/fidget/issues/133
+pkg "fidget" # pending https://github.com/treeform/fidget/issues/133
```
Comment (ringabout) [inline testament/important_packages.nim]:
```suggestion
pkg "fidget"
```

**Pull request merged**